pkgs/development/libraries/science/math/liblapack/default.nix

```diff
@@ -30,7 +30,11 @@ stdenv.mkDerivation rec {
     "-DCBLAS=ON"
     "-DBUILD_TESTING=ON"
   ] ++ lib.optional shared "-DBUILD_SHARED_LIBS=ON"
-    ++ lib.optional blas64 "-DBUILD_INDEX64=ON";
+    ++ lib.optional blas64 "-DBUILD_INDEX64=ON"
+    # Tries to run host platform binaries during the build
+    # Will likely be disabled by default in 3.12, see:
+    # https://github.com/Reference-LAPACK/lapack/issues/757
+    ++ lib.optional (stdenv.buildPlatform != stdenv.hostPlatform) "-DTEST_FORTRAN_COMPILER=OFF";
```

Comment:
```suggestion
    ++ lib.optional (stdenv.buildPlatform.canExecute stdenv.hostPlatform) "-DTEST_FORTRAN_COMPILER=OFF";
```

Reply:
I fixed this and updated #204875 as well.